ads/README.md

```diff
@@ -279,9 +279,13 @@ documentation with the new behaviors on user's consent choices. You can refer to
 
 #### JS reuse across iframes
 
-To allow ads to bundle HTTP requests across multiple ad units on the same page the object `window.context.master` will contain the window object of the iframe being elected master iframe for the current page. The `window.context.isMaster` property is `true` when the current frame is the master frame.
+To allow ads to bundle HTTP requests across multiple ad units on the same page, use `window.context.coordinator` to access the coordinating iframe window object, and `window.context.isCoordinator` to check if the current frame is the coordinating frame.
```

Comment:
@powerivq do you know if 3rd party code would need to make changes for this?

Reply:
They will. The best we can do is to offer an alias therefore.